HeartbeatKit/src/event-handler.js

```diff
@@ -210,6 +215,59 @@ EventHandler.prototype.logEvent = function(event) {
     }
 };
 
+var getAdobeMetadataKeys = function(attributes, Heartbeat) {
+    var AdobeMetadataLookupTable = {
+        // Ad Meta Data
+        ad_content_advertiser: Heartbeat.AdMetadataKeys.ADVERTISER,
+        ad_content_campaign: Heartbeat.AdMetadataKeys.CAMPAIGN_ID,
+        ad_content_creative: Heartbeat.AdMetadataKeys.CREATIVE_ID,
+        ad_content_placement: Heartbeat.AdMetadataKeys.PLACEMENT_ID,
+        ad_content_site_id: Heartbeat.AdMetadataKeys.SITE_ID,
+        ad_content_creative_url: Heartbeat.AdMetadataKeys.CREATIVE_URL,
+
+        // Audio Meta
+        content_artist: Heartbeat.AudioMetadataKeys.ARTIST,
+        content_album: Heartbeat.AudioMetadataKeys.ALBUM,
+        content_label: Heartbeat.AudioMetadataKeys.LABEL,
+        content_author: Heartbeat.AudioMetadataKeys.AUTHOR,
+        content_station: Heartbeat.AudioMetadataKeys.STATION,
+        content_publisher: Heartbeat.AudioMetadataKeys.PUBLISHER,
+
+        // Video Meta
+        content_show: Heartbeat.VideoMetadataKeys.SHOW,
+        stream_format: Heartbeat.VideoMetadataKeys.STREAM_FORMAT,
+        content_season: Heartbeat.VideoMetadataKeys.SEASON,
+        content_episode: Heartbeat.VideoMetadataKeys.EPISODE,
+        content_asset_id: Heartbeat.VideoMetadataKeys.ASSET_ID,
+        content_genre: Heartbeat.VideoMetadataKeys.GENRE,
+        content_first_air_date: Heartbeat.VideoMetadataKeys.FIRST_AIR_DATE,
+        content_digital_date: Heartbeat.VideoMetadataKeys.FIRST_DIGITAL_DATE,
+        content_rating: Heartbeat.VideoMetadataKeys.RATING,
+        content_originator: Heartbeat.VideoMetadataKeys.ORIGINATOR,
+        content_network: Heartbeat.VideoMetadataKeys.NETWORK,
+        content_show_type: Heartbeat.VideoMetadataKeys.SHOW_TYPE,
+
+        // TODO: This key is not represented in our docs
+        // ad_load: Heartbeat.VideoMetadataKeys.AD_LOAD,
+
+        content_mvpd: Heartbeat.VideoMetadataKeys.MVPD,
+        content_authorized: Heartbeat.VideoMetadataKeys.AUTHORIZED,
+        content_daypart: Heartbeat.VideoMetadataKeys.DAY_PART,
+        content_feed: Heartbeat.VideoMetadataKeys.FEED
+    };
+
+    var adobeMetadataKeys = {};
+    for (var attribute in attributes) {
+        var key = attribute;
+        if (attribute in AdobeMetadataLookupTable) {
```

Comment:
since `for in` is es6 and we don't have a babel compiler in our integrations, `for in` wouldn't be compatible with older browsers.  we should probably add to our bug bash an item for adding babel compiler to our integrations. but for now:

```suggestion
        if (AdobeMetadataLookupTable[attribute]) {
```